endpoint_auth_api_key/models/endpoint_mixin.py

```diff
@@ -20,9 +20,9 @@ def _selection_auth_type(self):
         return super()._selection_auth_type() + [("api_key", "API key")]
 
     def _validate_request(self, request):
-        super()._validate_request(request)
```

Comment:
careful with this type of change, by doing that you are changing the checks order
if you need a return to satisfy pre-commit, just add a final return